graphql/types.go

```diff
@@ -135,10 +135,13 @@ type SelectionSet struct {
 // (representing the name to be used in the output), args id: 166 (representing
 // arguments passed to the source field to be queried), and subselection name
 // representing the information to be queried from the resulting object.
+// The Parsed flag is used to make sure the args for this Selection are only
+// parsed once.
 type Selection struct {
 	Name         string
 	Alias        string
 	Args         interface{}
+	Parsed       bool
```

Comment:
should we restrict access to this variable?

also, could you co-locate the comment above closer to the field?

Reply:
Ah, package-private i.e. `parsed`? Seems reasonable to me

Yes, will move comment!